enterprise/internal/campaigns/syncer.go

```diff
@@ -149,6 +149,15 @@ func nextSync(h campaigns.ChangesetSyncData) time.Time {
 
 	// Simple linear backoff for now
 	diff := lastSync.Sub(lastChange)
+
+	// If the last change has happened SINCE our last sync this indicates a webhook
```

Comment:
Hmmm, is this what you mean?

```suggestion
	// If the last change has happened AFTER our last sync this indicates a webhook
```